styles/cryb/catppuccin.user.css

```diff
@@ -0,0 +1,359 @@
+/* ==UserStyle==
+@name Cryb Catppuccin
+@namespace github.com/catppuccin/userstyles/styles/cryb
+@homepageURL https://github.com/catppuccin/userstyles/tree/main/styles/cryb
+@version 0.1.0
+@updateURL https://github.com/catppuccin/userstyles/raw/main/styles/cryb/catppuccin.user.css
+@supportURL https://github.com/catppuccin/userstyles/issues?q=is%3Aopen+is%3Aissue+label%3Acryb
+@description Soothing pastel theme for Cryb
+@author Catppuccin
+@license MIT
+
+@preprocessor less
+@var select lightFlavor "Light Flavor" ["latte:Latte*", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha"]
+@var select darkFlavor "Dark Flavor" ["latte:Latte", "frappe:Frappé", "macchiato:Macchiato", "mocha:Mocha*"]
+@var select accentColor "Accent" ["rosewater:Rosewater", "flamingo:Flamingo", "pink:Pink", "mauve:Mauve*", "red:Red", "maroon:Maroon", "peach:Peach", "yellow:Yellow", "green:Green", "teal:Teal", "blue:Blue", "sapphire:Sapphire", "sky:Sky", "lavender:Lavender", "subtext0:Gray"]
+==/UserStyle== */
+
+@-moz-document domain('0.0.0.0:port') {
```

Comment:
How does one typically access this? When @isabelroses mentioned `0.0.0.0:port` in the Discord she meant more like https://github.com/catppuccin/userstyles/blob/93914ea87e2efc1d4f6f47c6419397793917275a/styles/homepage/catppuccin.user.css#L18 I think. Is there a localhost port Cryb usually uses? Is there a public URL? Etc.

Reply:
Like I've mentioned previously in the Discord, there's currently no public instances of Cryb (as far as I'm aware). Although I've gone ahead and changed the domain to a placeholder as you've requested.

Is that enough to solve the issue, or do you require a public instance of Cryb in order to review/pass the userstyle?

Reply:
I'll have to figure out how to set it up locally before we can merge, so having a public instance sure helps get this through faster.